@!!BEGIN
@!!type:obligationsnumber
@!!count:0
@!!END

File "/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library/TLAPS.tla", line 1, character 1 to line 363, character 77:
[INFO]: All 0 obligation proved.
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:1
@!!END

** Unexpanded symbols: ---

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:being proved
@!!prover:smt
@!!meth:time-limit: 5
@!!obl:
0 \in 1

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:failed
@!!prover:smt
@!!meth:time-limit: 5; time-used: 5.0 (100%)
@!!reason:timeout
@!!already:false
@!!obl:
0 \in 1

@!!END

Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:failed
@!!prover:zenon
@!!meth:time-limit: 10; time-used: 0.0 (0%)
@!!reason:false
@!!already:false
@!!obl:
0 \in 1

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:failed
@!!prover:isabelle
@!!meth:auto; time-limit: 30
@!!reason:Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin

@!!already:false
@!!obl:
0 \in 1

@!!END

File "./zero_in_one_test.tla", line 13, characters 5-11:
[ERROR]: Could not prove or check:
           0 \in 1
File "./zero_in_one_test.tla", line 1, character 1 to line 15, character 4:
[ERROR]: 1/1 obligation failed.
There were backend errors processing module `"zero_in_one_test"`.
